docs/getting-started-guides/coreos/bare_metal_offline.md

```diff
@@ -0,0 +1,646 @@
+# Bare Metal CoreOS with Kubernetes (OFFLINE)
+Deploy a CoreOS running Kubernetes environment. This particular guild is made to help those in an OFFLINE system, wither for testing a POC before the real deal, or you are restricted to be totally offline for your applications.
+
+
+## Goal
+At this point in the community we can not and should not try and solve any of the current issues at hand in the Docker space. 
+Instead, try to add ease of use for enterprise customers looking to develop or deploy applications into the new container ecosystem offline.
```

Comment:
Assume the reader of the getting started guide has already decided to try kubernetes.  Therefore, you can delete the above two lines.